build-system/tasks/check-links.js

```diff
@@ -137,6 +137,10 @@ function filterWhitelistedLinks(markdown) {
   // Dailymotion link that is marked dead in Travis CI
   filteredMarkdown = filteredMarkdown.replace(/https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
```

Comment:
While you're making this change, would you mind breaking this long line into two by adding a new line after `replace(`?